data-collection/src/main/java/refactoringml/ProcessMetricTracker.java

```diff
@@ -74,6 +77,29 @@ public void resetCounter(CommitMetaData commitMetaData) {
 
 	public ProcessMetrics getCurrentProcessMetrics() { return currentProcessMetrics; }
 
+	public int getCommitCountThreshold() { return currentCommitThreshold; }
+
+	//Filter class files that were not refactored in the last K commits and not already found with a lower K.
+	//TODO: If a class has an inner class only one instance is stored in the database
+	// The fix (>= instead of > : currentCommitThreshold) leads to stable commit duplicates in the DB in case of multiple refactorings in the current commit
+	public boolean calculateStability(List<Integer> commitThresholds){
+		for (Integer threshold : commitThresholds){
+			//1. Test if the class file was not refactored for the last K commits
+			if(isStable(threshold) &&
+					//2. Avoid duplicates:
```

Comment:
maybe also extract these two conditions (i.e., `isStable(threshold)` and `(threshold > currentCommitThreshold || threshold == getCommitCounter())`) in two variables, each one with its own comments, so that we don't have to break the `if` in this strange manner just to write the comments.